server-auth/saml/build.gradle

```diff
@@ -1,4 +1,8 @@
 dependencies {
     implementation project(':server')
-    implementation libs.armeria.saml
+    implementation(libs.armeria.saml) {
+        exclude group: 'org.bouncycastle'
+    }
+
+    implementation libs.bouncycastle.bcprov
```

Comment:
nit; I think it should be possible to add a `strictly` since this is the only version that will be used across all modules

Reply:
Yeah, we can probably add it later. 😉 
But, the current problem is that they are using different artifacts.
e.g. `org.bouncycastle:bcprov-jdk15on` and `org.bouncycastle:bcprov-jdk18on`

Reply:
I've added the module as well in order to be more specific. 😉